README.md

```diff
@@ -29,7 +29,7 @@
 * Both strict or loose modes of failover to be sure that everything will either
   fail on any error or try to complete, no matter of what.
 
-* Interactive password authentication as well as SSH public key authentication.
+* Interactive password authentication as well as SSH public key authentication. Will use ssh-agent if present. On Windows, orgalorg can connect to **pageant** or **openssh agent**.
```

Comment:
Also, could you wrap the line to 80 symbols since the entire README.md file does it already?